docs/source/tutorials/datasets.rst

```diff
@@ -0,0 +1,457 @@
+""""""""""""""""
+
+Use the method :meth:`create_image_group() <.EncordClientDataset.create_image_group>` to upload images and create an image group using |company| storage.
+
+.. autolink-concat:: section
+
+.. autolink-preface::
+
+    from encord.client import EncordClientDataset
+    dataset_client = EncordClientDataset()  #  user_client.get_dataset_client("<dataset_hash>")
+
+.. code-block:: python
+
+    dataset_client.create_image_group(
+        [
+            "path/to/your/img1.jpeg",
+            "path/to/your/img2.jpeg",
+        ]
+    )
+
+This will upload the given list of images to the dataset associated with the :class:`dataset_client <.EncordClientDataset>` and create an image group.
+
+.. note::
+
+    Image groups are images of the same resolution, so if ``img1.jpeg`` and ``img2.jpeg`` from the example above are of shape ``[1920, 1080]`` and ``[1280, 720]``, respectively, they will end up in each of their own image group.
+
+
+.. note::
+
+    Images in an image group will be assigned a ``data_sequence`` number, which is based on the order or the files listed in the argument to :meth:`create_image_group <.EncordClientDataset.create_image_group>` above.
+    If the ordering is important, make sure to provide a list with filenames in the correct order.
+
+
+Adding data from a private cloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#.  Use :meth:`user_client.get_cloud_integrations() <.EncordUserClient.get_cloud_integrations>` method to retrieve a list of available Cloud Integrations
+#.  Grab the id from the integration of your choice and call :meth:`dataset_client.add_private_data_to_dataset() <.EncordClientDataset.add_private_data_to_dataset>` on the ``dataset_client`` with either the absolute path to a json file or a python dictionary in the format specified in the :xref:`private_cloud_section` of the |platform| datasets documentation
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/datasets/add_private_data.py
+            :language: python
+
+
+..
+    TODO: Some things are missing here:
+        1. A link to where cloud integrations are enabled on the web-app
+        2. A way to enable cloud integrations from the SDK?
+        3. Sample outputs of the given script
+
+
+
+Deleting data
+-------------
+
+You can remove both Videos and Image Groups from datasets created using both the |platform| and the |sdk|.
+Use the method :meth:`dataset_client.delete_data() <.EncordClientDataset.delete_data>` to delete from a dataset.
+
+.. autolink-concat:: section
+
+.. autolink-preface::
+
+    from encord.client import EncordClientDataset
+    dataset_client = EncordClientDataset()
+
+.. code-block:: python
+
+    dataset_client.delete_data(
+        [
+            "<video1_data_hash>",
+            "<image_group1_data_hash>",
+        ]
+    )
+
+
+In case the Video or Image Group belongs to |company|-hosted storage, the corresponding file shall be removed from the Encord-hosted storage.
```

Comment:
```suggestion
In case the Video or Image Group belongs to |company|-hosted storage, the corresponding file will be removed from the Encord-hosted storage.
```